### DISCUSSION TITLE
Unable to save changes while in devcontainer

### DISCUSSION THREAD

**Discussion by akstamimech** [tudelft/AE4353-Y24#10] [Ex_1]
Hi, 
I am having an issue saving an edited file while in a devcontainer. The error I am facing is as below. I have tried using chmod to give myself permission to edit the documents and the folders, unfortunately this doesnt seem to work. What could be the solution?

Failed to save '__init__.py': Unable to write file 'vscode-remote://dev-container+7b22686f737450617468223a225c5c5c5c77736c2e6c6f63616c686f73745c5c5562756e74752d32302e30345c5c726f6f745c5c4145343335332d593234222c226c6f63616c446f636b6572223a66616c73652c22636f6e66696746696c65223a7b22246d6964223a312c2270617468223a222f726f6f742f4145343335332d5932342f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c22736368656d65223a227673636f64652d66696c65486f7374227d7d/workspaces/AE4353-Y24/ex_1/__init__.py' (NoPermissions (FileSystemError): Error: EACCES: permission denied, open '/workspaces/AE4353-Y24/ex_1/__init__.py')

**Comment by Huizerd** [comment]
if you open a terminal anywhere, as what user does this happen? as root or your own user? if root, it could be that the repository was cloned as root (and therefore can only accessed by root). in combination with your user not being added to the docker group this could explain the error. please come and see us during the session today and we'll have a look!
